apps/mobile/app/(app)/debt/[id]/extra.tsx

```
@@ -0,0 +1,197 @@
+                    {daysSooner > 0 ? `Debt-free ${daysSooner} ${daysSooner === 1 ? 'day' : 'days'} sooner` : 'Same debt-free month'}
+                  </Num>
+                </View>
+                <Muted className="mt-2">
+                  New date: <Num className="text-[13px] text-muted">{clearsEverything ? 'today' : monthYear(monthFromNow(previewOutcome!.months))}</Num>
+                  {interestSaved !== null && interestSaved > 0 ? ` · saves ${money(interestSaved)} in interest` : ''}
+                </Muted>
+                {daysSooner === 0 ? (
+                  <Muted className="mt-1">Still worth it — every dollar of principal cuts interest, even inside the same month.</Muted>
+                ) : null}
+                {clearsEverything ? (
+                  <Body className="mt-2 font-semibold text-emerald-700">This clears your last debt. You'd be debt-free today. 🎉</Body>
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&`#39`;`, `&rsquo;`.

(react/no-unescaped-entities)